marketing-guidelines.md

```diff
@@ -0,0 +1,99 @@
+# OpenTelemetry Marketing Guidelines for Contributing Organizations
+
+OpenTelemetry is a collaboration among end-users, adjacent OSS projects, and
+vendors who ultimately sell products and services built upon OpenTelemetry data
+or components. Like many standards-oriented projects, the vendors that partner
+on OpenTelemetry also compete in the marketplace, and for this reason it’s
+important to establish some ground rules and expectations for how contributing
+organizations communicate and message about OpenTelemetry.
+
+In fact, OpenTelemetry’s success depends on both the reality and the perception
+of sincere collaboration between the many parties (and vendors) involved.
+There’s a lot of superb technical work happening within OTel, and we want to
```

Comment:
Let's use OpenTelemetry or have (a.k.a. OTel) somewhere before in that document.